packages/gatsby-plugin-emotion/src/wrap-element.js

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+import { cache } from "emotion"
+import { CacheProvider } from "@emotion/core"
+
+export const wrapElement = element => (
+  <CacheProvider value={cache}>{element}</CacheProvider>
```

Comment:
This is necessary if people are using `injectGlobal`, `keyframes`, `css`, etc. from the `emotion` package, which seems likely.